originations/domain/risk_segment_assigner/risk_segment_calculator.py

```diff
@@ -29,7 +29,7 @@ async def risk_pricing_assigner(
             "reference_id": str(request.application_id),
             "risk_score": risk_score,
             "risk_segment": risk_segment,
-            "price": apr,
+            "price": nullable("double", apr),
```

Comment:
Need to use specific formatting for nullable AVROs when using JSON-encoding, so built a helper function to achieve this.